js/buildOptions.py

```diff
@@ -155,6 +159,11 @@ def parseShellOptions(inputArgs):
     if sps.isMac:
         buildOptions.buildWithClang = True  # Clang seems to be the only supported compiler
 
+    # vTune and profiling is broken for Ubuntu due to Ubuntu's specific glibc/PLT
```

Comment:
Actually this blob can be removed once :sstangl's follow-up patch in bug 1339190 lands.